hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/audit/parser/common/DatabaseHelper.java

```diff
@@ -142,27 +156,27 @@ private static boolean insertAudits(String dbName, String logs) {
   }
 
   private static ArrayList<AuditEntry> parseAuditLogs(String filePath)
-      throws Exception {
-    ArrayList<AuditEntry> listResult = new ArrayList<AuditEntry>();
-    try(FileInputStream fis = new FileInputStream(filePath);
-        InputStreamReader isr = new InputStreamReader(fis, "UTF-8");
+      throws IOException {
+    ArrayList<AuditEntry> listResult = new ArrayList<>();
+    try (FileInputStream fis = new FileInputStream(filePath);
+        InputStreamReader isr = new InputStreamReader(fis, UTF_8);
         BufferedReader bReader = new BufferedReader(isr)) {
-      String currentLine = null;
-      String[] entry = null;
+      String currentLine;
+      String[] entry;
       AuditEntry tempEntry = null;
-      String nextLine = null;
+      String nextLine;
       currentLine = bReader.readLine();
       nextLine = bReader.readLine();
```

Comment:
Can you please also join the declaration and assignment (of `currentLine` and `nextLine`)?

Reply:
Done.